### DISCUSSION TITLE
'Issues' on the docker project 'https://github.com/Keyfactor/ejbca-containers' not available

### DISCUSSION THREAD

**Discussion by kuehne-trustable-de** [Keyfactor/ejbca-ce#236] [Q&A]
Despite its mentioning in the Readme the issues option is not available on the project 'https://github.com/Keyfactor/ejbca-containers'. The link redirects to the 'Pull requests'

**Comment by primetomas** [comment]
Thanks. I edited the page to point to EJBCA Issues instead.
